### ISSUE TITLE
Could not find  the game config folder

### ISSUE THREAD

**Issue by bethakneemiller** [MestreLion/hackmatch#6]
![Screenshot 2023-11-04 203451](https://github.com/MestreLion/hackmatch/assets/131637347/d3c57cc8-8056-43b9-ada3-a68f5fda1f16)

Playing the Windows version and get this error every time I try to launch the game.  No idea what to do from here.

**Comment by pnemes** [comment]
I've had to explicitly supply my steam user id in
`config["steam_user_id"]`
on windows for it to work

**Comment by MestreLion** [comment]
Weird, I wonder why the glob does not work in Windows

**Comment by MestreLion** [comment]
> I've had to explicitly supply my steam user id in `config["steam_user_id"]` on windows for it to work

And since I was to lazy to implement an actual config file, you have to edit the source code to supply it. Ouch.

A workaround would be to add it as a command-line argument, something like `--steam-user-id xxxxx`. That would be easier to implement than a config file

**Comment by pnemes** [comment]
I am surprised as well, since it is a python function so the globexpansion
is not done by the shell,
and definitely not by the called system function. weird

**Comment by nitrrose** [comment]
I'm having the same issue with my installation, and frankly I don't understand how to do anything that's been said- x.x
Can somebody dumb it down for me lol
